src/main/java/io/github/externschool/planner/service/ScheduleServiceImpl.java

```diff
@@ -57,6 +57,11 @@ public ScheduleServiceImpl(final ScheduleEventRepository eventRepository,
      * @return ScheduleEvent
      */
     @Deprecated
+    @Override
```

Comment:
i's a very BAD idea to add new code between an existing method and its annotation: now YOUR method is deprecated not the old one.